mmcv/runner/hooks/logger/wandb.py

```diff
@@ -87,6 +93,17 @@ def before_run(self, runner):
     @master_only
     def log(self, runner):
         tags = self.get_loggable_tags(runner)
+
+        # log best metric
+        for k, v in list(tags.items()):
```

Comment:
Single letter variable is hard to understand, could you rename k(tag_keys), v(tag_values) bk(metric_keys) and bv(best_metric_value)?